lib/javascripts/external_assets.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Could you update the README [here](https://github.com/zendesk/app_scaffold/blob/master/README.md#external-dependencies) pls?